imports/blocks/change-payments/index.js

```diff
@@ -1,45 +1,52 @@
-import { Component, PropTypes } from "react";
+// @flow
+
+import { Component } from "react";
 import { connect } from "react-redux";
 
-import { modal, give as giveActions } from "../../store";
+import {
+  modal,
+  give as giveActions,
+} from "../../store";
 
-import AccountType from "../../components/accountType";
+import Layout from "./Layout";
 
 const map = (state) => ({
   savedAccount: state.give.savedAccount,
 });
-@connect(map)
-export default class ChangePayments extends Component {
 
-  static propTypes = {
-    dispatch: PropTypes.func,
-    savedAccounts: PropTypes.array, // eslint-disable-line
-    currentAccount: PropTypes.object, // eslint-disable-line
-  }
+export const withRedux = connect(map);
+
+type IChangePayments = {
+  dispatch: Function,
+  savedAccounts: Object[],
+  currentAccount: Object,
+};
+
+type IChangePaymentsState = {
+  savedAccount: Object,
+}
+
+export class ChangePayments extends Component {
+  props: IChangePayments;
+  state: IChangePaymentsState;
 
   state = {
     savedAccount: null,
   }
 
-  changePaymentHeader = () => (
-    <h4 className="text-center flush-bottom">
-      Change Payment Account
-    </h4>
-  );
-
-
-  changeAccounts = (e) => {
+  changeAccounts = (e: Event) => {
     e.preventDefault();
 
     this.props.dispatch(giveActions.setAccount(this.state.savedAccount));
     this.props.dispatch(modal.hide());
   }
 
-  choose = (e) => {
+  choose = (e: SyntheticInputEvent) => {
     e.preventDefault();
 
     const { id } = e.currentTarget;
-    let act = {};
+    // XXX This is wrong, and needs to be changed...eventaully
```

Comment:
@jbaxleyiii When working through this with @johnthepink we discussed that `any` isn't the best solution long term for the typed definition here.